src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -63,6 +73,8 @@ private static void setupDrivers(Application application) {
 		DriverFeature.addDriver("Line Simulator", driver);
 		DriverFeature.getDriverManager().setCurrentDriver(driver);
 
+		MacroFeature.getMacroDriver().setPreviousDriver(driver);
```

Comment:
Think about using "Observer for a driver changes events" feature (in future update)